models/staging/stg_dbt__artifacts.sql

```diff
@@ -36,6 +36,8 @@ artifacts as (
     select
         command_invocation_id,
         dbt_cloud_run_id,
+        -- This ID provides a reliable ID, regardless of whether running in a local or cloud environment.
+        sha2_hex(coalesce(dbt_cloud_run_id, command_invocation_id), 256) as artifact_run_id,
```

Comment:
@NiallRees - this is the bit that I'd love your view on.

Reply:
Great idea. Linked issue: https://github.com/brooklyn-data/dbt_artifacts/issues/53